.github/workflows/deploy-latest-helm-version.yaml

```diff
@@ -10,6 +10,10 @@ jobs:
   update_helm_releases:
     runs-on: ubuntu-latest
     steps:
+      - name: Sleep for 3 minutes to allow time for the helm package to be available
```

Comment:
ouch. so the package doesn't become available on the helm registry for a bit?

Reply:
i think it just takes a minute or two